R/slide.R

```diff
@@ -88,62 +88,85 @@
 #'     before = 1, as_list_col = TRUE
 #'   ) %>%
 #'   ungroup()
-epi_slide <- function(x, f, ..., before = NULL, after = NULL, ref_time_values = NULL,
-                      new_col_name = "slide_value", as_list_col = FALSE,
-                      names_sep = "_", all_rows = FALSE) {
+epi_slide <- function(
+    x, f, ...,
```

Comment:
suggest: also dot-prefix `x` and `f`, like in `purrr::map`

(maybe we lose some ease in searchability since there's .x into `epi_slide` and `.x` the pronoun-like thing in formulas and tidyeval, but that's shared by `purrr::map`)

Reply:
sure yea, prefixing them sounds reasonable.

(huh, interesting, well intuitively most of my uses of purrr::map it's piped into, so that takes care of the first argument. using purrr::map with a do.call type construct with list(.x=...) seems really unlikely.)